Sources/ShopifyCheckoutSheetKit/CheckoutBridge.swift

```diff
@@ -119,6 +120,12 @@ extension CheckoutBridge {
 				let webPixelsDecoder = WebPixelsEventDecoder()
 				let event = try webPixelsDecoder.decode(from: container, using: decoder)
 				self = .webPixels(event: event)
+			case "payButtonTouch":
+				if #available(iOS 10, *) {
+					let impactMed = UIImpactFeedbackGenerator(style: .medium)
+					impactMed.impactOccurred()
+				}
```

Comment:
Not sure this is the best place for this code. It feels like an implementation detail, whereas this bridge mainly acts as a controller for event messages